src/applications/mhv-medical-records/containers/VitalDetails.jsx

```diff
@@ -122,11 +122,19 @@ const VitalDetails = props => {
   useEffect(
     () => {
       if (records?.length) {
-        focusElement(document.querySelector('h2'));
         paginatedVitals.current = paginateData(records);
         setCurrentVitals(paginatedVitals.current[currentPage - 1]);
       }
     },
+    [records],
+  );
+
+  useEffect(
+    () => {
+      if (currentPage > 1 && records?.length) {
+        focusElement(document.querySelector('h2'));
+      }
```

Comment:
I notice this block doesn't contain `window.scrollTo({ top: 0, left: 0, behavior: 'smooth' })` like it does in the RecordList version. Does it need to be added here? Maybe a question for Bobby.

Reply:
Added. Will meet with Bobby as well to discuss.